app/models/volunteer_application.rb

```diff
@@ -0,0 +1,21 @@
+class VolunteerApplication < ActiveRecord::Base
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  VALID_PHONE_REGEX = /[0-9]{10}/
+  before_save :downcase_email
```

Comment:
You don't need to change this, but I wanted to warn you about the dangers of `before_save`. It's often the source of unexpected behavior because it's very difficult to save without `before_save`s triggering. I tend to never use `before_save` unless I have to. I think it's fine here, but I'd always suggest trying to do something else first before adding one. Same with `before_create`, `before_validation`, etc.